packages/x-charts/src/models/seriesType/scatter.ts

```diff
@@ -1,7 +1,14 @@
 import { DefaultizedProps } from '../helpers';
-import { CartesianSeriesType, CommonDefaultizedProps, CommonSeriesType } from './common';
+import { CartesianSeriesType, CommonDefaultizedProps, CommonSeriesType, SeriesId } from './common';
 
-export type ScatterValueType = { x: number; y: number; id: string | number };
+export type ScatterValueType = {
+  x: number;
+  y: number;
+  /**
+   * A unique id identifying the scatter point.
```

Comment:
nit to be consistent

```suggestion
   * A unique identifier for the scatter point.
```